x-pack/plugins/ml/public/embeddables/single_metric_viewer/single_metric_viewer_embeddable_factory.tsx

```diff
@@ -88,6 +88,32 @@ export const getSingleMetricViewerEmbeddableFactory = (
 
       const api = buildApi(
         {
+          isEditingEnabled: () => true,
+          getTypeDisplayName: () =>
+            i18n.translate('xpack.ml.singleMetricViewerEmbeddable.typeDisplayName', {
+              defaultMessage: 'Single metric viewer',
```

Comment:
This string is appended to `Edit...`
```suggestion
              defaultMessage: 'single metric viewer',
```

Reply:
Updated in 7e3b004a5255f450ce23964cfa2235a68126dcf5